main.js

```diff
@@ -532,9 +551,9 @@ function scrollToTrack(trackId) {
 
 fetch(new URL('/player', location))
 .then(resp => resp.json())
-.then(playerData => {
-  createPlayerMarker(playerData);
-  map.setView(playerData.position, initialZoom)
+.then(data => {
+  updatePlayerOverlays(data);
+  map.setView(data.players[0].position, initialZoom);
```

Comment:
As above, I'd suggest trying to find `player` for single-player, and if not present, just leave the map at the initial view to show the whole map.